sky/jobs/utils.py

```diff
@@ -672,11 +770,18 @@ def get_hash(task):
             status_counts[managed_job_status.value] += 1
 
     columns = [
-        'ID', 'TASK', 'NAME', 'RESOURCES', 'SUBMITTED', 'TOT. DURATION',
-        'JOB DURATION', '#RECOVERIES', 'STATUS'
+        'ID',
+        'TASK',
+        'NAME',
+        'RESOURCES',
+        'SUBMITTED',
+        'TOT. DURATION',
+        'JOB DURATION',
+        '#RECOVERIES',
+        'STATUS',
     ]
     if show_all:
-        columns += ['STARTED', 'CLUSTER', 'REGION', 'FAILURE']
+        columns += ['STARTED', 'CLUSTER', 'REGION', 'FAILURE', 'SCHED. STATE']
```

Comment:
nit: I would prefer to not have the `sched. state` column, instead, we may want to do something similar as `kubectl describe pod` where it shows detailed description of what the pod is working on in the same state. For example, we can maybe rename the `FAILURE` column to be `DESCRIPTION`.

Reply:
Don't want to spend too much time on this but I'll take a look.

Reply:
Yep, don't need to be a large change. Just adding the state as a description in the `FAILURE` column (now should rename to `DESCRIPTION`